contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
```

Comment:
To simplify the example contract, explanations of conventions such as where to use CamelCase can be put in the style guide itself.  Then the contract can simply demonstrate the scenarios in which the style guide is being correctly used.  I believe the dev tag is meant to designate comments which might help other developers use your code.  i.e. if there is some limitation to the function parameters or some rule about how something must be invoked.